test/break-all-the-rules.md

```diff
@@ -105,6 +105,11 @@ Strong **with** different style {MD050}
 <!-- markdownlint-disable-next-line MD053 -->
 [url]: https://example.com/page
 
+| table | text |
```

Comment:
Wow, good catch, most folks miss this file. :) To be a nitpicker, just a single violation is all that's needed.